code/datums/supplypacks.dm

```diff
@@ -38,2607 +27,1468 @@ GLOBAL_LIST_INIT(all_supply_groups, list("Operations", "Weapons", "Hardpoint Mod
-	containername = "\improper MBX 900 ammo crate"
-	group = "Ammo"
 
-/datum/supply_packs/ammo_rpg_regular
-	name = "M5 RPG HE rockets crate (x6)"
-	contains = list(
-					/obj/item/ammo_magazine/rocket,
-					/obj/item/ammo_magazine/rocket,
-					/obj/item/ammo_magazine/rocket,
-					/obj/item/ammo_magazine/rocket,
-					/obj/item/ammo_magazine/rocket,
-					/obj/item/ammo_magazine/rocket
-					)
-	cost = 40
-	containertype = /obj/structure/closet/crate/explosives
-	containername = "\improper regular M5 RPG ammo crate"
-	group = "Ammo"
-
-/datum/supply_packs/ammo_rpg_ap
-	name = "M5 RPG AP rockets crate (x6)"
-	contains = list(
-					/obj/item/ammo_magazine/rocket/ap,
-					/obj/item/ammo_magazine/rocket/ap,
-					/obj/item/ammo_magazine/rocket/ap,
-					/obj/item/ammo_magazine/rocket/ap,
-					/obj/item/ammo_magazine/rocket/ap,
-					/obj/item/ammo_magazine/rocket/ap
-					)
-	cost = 40
-	containertype = /obj/structure/closet/crate/explosives
-	containername = "\improper armor piercing M5 RPG ammo crate"
-	group = "Ammo"
-
-/datum/supply_packs/ammo_rpg_wp
-	name = "M5 RPG WP rockets crate (x6)"
-	contains = list(
-					/obj/item/ammo_magazine/rocket/wp,
-					/obj/item/ammo_magazine/rocket/wp,
-					/obj/item/ammo_magazine/rocket/wp,
-					/obj/item/ammo_magazine/rocket/wp,
-					/obj/item/ammo_magazine/rocket/wp,
-					/obj/item/ammo_magazine/rocket/wp
-					)
-	cost = 30
-	containertype = /obj/structure/closet/crate/explosives
-	containername = "\improper white phosphorus M5 RPG ammo crate"
-	group = "Ammo"
-
-/datum/supply_packs/ammo_smartgun
-	name = "M56 smartgun powerpack crate (x5)"
-	contains = list(
-					/obj/item/smartgun_powerpack,
-					/obj/item/smartgun_powerpack,
-					/obj/item/smartgun_powerpack,
-					/obj/item/smartgun_powerpack,
-					/obj/item/smartgun_powerpack
-					)
-	cost = 60
-	containertype = /obj/structure/closet/crate/ammo
-	containername = "\improper smartgun powerpack crate"
-	group = "Ammo"
-
-/datum/supply_packs/ammo_smartmachinegun
-	name = "T26 smartmachinegun ammo crate (x3)"
-	contains = list(
-					/obj/item/ammo_magazine/standard_smartmachinegun,
-					/obj/item/ammo_magazine/standard_smartmachinegun,
-					/obj/item/ammo_magazine/standard_smartmachinegun
-					)
-	cost = 30
-	containertype = /obj/structure/closet/crate/ammo
-	containername = "\improper T-26 ammo crate"
-	group = "Ammo"
+/datum/supply_packs/ammo/rpg_regular
+	name = "M5 RPG HE rocket"
+	contains = list(/obj/item/ammo_magazine/rocket)
+	cost = 7
+
+/datum/supply_packs/ammo/rpg_ap
+	name = "M5 RPG AP rocket"
+	contains = list(/obj/item/ammo_magazine/rocket/ap)
+	cost = 7
+
+/datum/supply_packs/ammo/rpg_wp
+	name = "M5 RPG WP rocket"
+	contains = list(/obj/item/ammo_magazine/rocket/wp)
+	cost = 5
+
+/datum/supply_packs/ammo/smartgun
+	name = "M56 smartgun powerpack"
+	contains = list(/obj/item/smartgun_powerpack)
+	cost = 12
+
+/datum/supply_packs/ammo/smartmachinegun
+	name = "T26 smartmachinegun ammo"
```

Comment:
name = "T29 smartmachinegun ammo"

Reply:
i guess whoever added it put the wrong numbers in